.gitignore

```
@@ -5,3 +5,6 @@
 
 # ignore the KiCad footprint cache
 fp-info-cache
+
+# ignore project backups
+*-backups*
```

According to https://docs.kicad.org/6.0/en/getting_started_in_kicad/getting_started_in_kicad.html#tutorial_part_1_project and my testing, it looks like project backups are placed in a directory that uses a naming scheme of `<project name>-backups`. Based on this, the following would be a more appropriate ignore rule:
```suggestion
*-backups/
```